jupyter_kernel_test/msgspec_v5.py

```diff
@@ -0,0 +1,336 @@
+"""Message schemas for message spec version 5"""
```

Comment:
This is excellent!  I think this and the `.json` file it emits should be in `nbformat`.

Reply:
I'd say `jupyter_client` would probably be the place for it - that's where the messaging protocol is defined. But I thought I'd put it here first, where there's a clear requirement for it.

Reply:
Ah, right, agreed!